### PR TITLE
When a request fails with an invalid FileMaker token, try to repeat t…

### PR DESCRIPTION
**Pull request by DASPRiD**
…he request with a new token

### CODE REVIEW

File: src/Client.ts [hunk 1]
```diff
@@ -55,6 +55,12 @@ export default class Client
 
         if (!response.ok) {
             const data = await response.json();
+
+            if (firstTry && data.messages[0].code === '952') {
+                this.token = null;
+                return this.request(path, request, false);
```
Comment (lart2150) [inline src/Client.ts:61]: This needs an await

**Review by lart2150 — CHANGES_REQUESTED** [review]
I tested this and it clears the session but returns a promise instead of the result from the second call.

File: src/Client.ts (hunk 1 above)
Comment (lart2150) [reply src/Client.ts:61]: I was wrong his his needing an await we need to do something like
request.headers.delete('Authorization'); before we call request a second time.

**Comment by DASPRiD** [comment]
This has been addressed in https://github.com/soliantconsulting/fm-data-api-client/commit/ef9b348ab7273e4c7a7b60840bb4a141d1557b43.

**Pull request closed without merging**